src/PIL/Image.py

```diff
@@ -206,6 +207,15 @@ class Quantize(IntEnum):
     RLE = core.RLE
     FIXED = core.FIXED
 
+Tile = NamedTuple(
+    "Tile",
+    [
+        ("encoder_name", str),
+        ("extents", Tuple[int, int, int, int]),
+        ("offset", int),
+        ("tile_args", Tuple),
+    ],
+)
```

Comment:
```suggestion
class Tile(NamedTuple):
    encoder_name: str
    extents: tuple[int, int, int, int]
    offset: int
    tile_args: tuple
```

And let's name it `_Tile` so it's not part of the public API?